editor/debugger/debug_adapter/debug_adapter_parser.cpp

```diff
@@ -600,12 +600,19 @@ Dictionary DebugAdapterParser::ev_continued() const {
 	return event;
 }
 
-Dictionary DebugAdapterParser::ev_output(const String &p_message) const {
+Dictionary DebugAdapterParser::ev_output(const String &p_message, RemoteDebugger::MessageType p_type) const {
 	Dictionary event = prepare_base_event(), body;
 	event["event"] = "output";
 	event["body"] = body;
 
-	body["category"] = "stdout";
+	switch (p_type) {
+		case RemoteDebugger::MessageType::MESSAGE_TYPE_ERROR:
+			body["category"] = "stderr";
+			break;
+		default:
+			body["category"] = "stdout";
+			break;
+	}
```

Comment:
```suggestion
	body["category"] = (p_type == RemoteDebugger::MessageType::MESSAGE_TYPE_ERROR) ? "stderr" : "stdout";
```
Since we're just using two cases I'd say a switch statement isn't needed here